README.md

```diff
@@ -1,29 +1,7 @@
 # **FreeCodeCamp**
 
-## Introduction to Advanced Node and Express Challenges
+## [Introduction to Advanced Node and Express Challenges](https://www.freecodecamp.org/learn/quality-assurance/advanced-node-and-express/)
```

Comment:
How do we want this @RandellDawson? I think we should add how specifically to have it in the issue. I'm fine with this, although I would make the `F` lowercase

Reply:
I don't think the `**FreeCodeCamp**` needs to be present at all honestly.  It would just be:
```md
# [Introduction to Advanced Node and Express Challenges](https://www.freecodecamp.org/learn/quality-assurance/advanced-node-and-express/)
```

Reply:
I don't really care either way, I do kind of like the freecodecamp in there. But we did kind of decide not to have it already, so maybe we should just do that. It's pretty minor, but if we can all agree on what we want, then we will know what to put on the rest of them.

What do you think @Sky020?

Reply:
@moT01 I go back and forth. It serves no use for the Camper, but outsiders could come across a project repo, and benefit.

If majority of the other boilerplate READMEs do not have it, I would remove this one and the rest.